devsetup/scripts/standalone.sh

```diff
@@ -80,7 +89,7 @@ export HOST_PRIMARY_RESOLV_CONF_ENTRY=${HOST_PRIMARY_RESOLV_CONF_ENTRY}
 export INTERFACE_MTU=${INTERFACE_MTU:-1500}
 export NTP_SERVER=${NTP_SERVER:-"clock.corp.redhat.com"}
 export EDPM_COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED:-true}
-export CEPH_ARGS="${CEPH_ARGS:--e \$HOME/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml}"
```

Comment:
I'm not sure if we'll ever need to do it, but I think that with this change we loose the ability to modify the ceph deployment. If I understand correctly, with this change we can only choose to not deploy ceph, or deploy it with `-e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml`.

Reply:
ugh, you are right. I misread that and thought it was always redefined in openstack.sh.
I can do a follow up to fix that? (Or if people insist I can change this PR)

Reply:
So ... I updated it to make it possible to modify the ceph deployment again. Good catch @cescgina++